cli/data.go

```diff
@@ -831,12 +824,10 @@ func writeData(writer *bufio.Writer, dataRow []byte, numWrites *uint64) error {
 	}
 
 	// Periodically flush to keep buffer size down.
-	if *numWrites == uint64(10_000) {
+	if uint64(writer.Size()) > uint64(10_000_000) {
```

Comment:
nit, no need for `uint64` here I think?

```suggestion
	if writer.Size() > 10_000_000 {
```